apps/src/templates/projects/projectsModule.js

```diff
@@ -26,4 +88,28 @@ export function selectGallery(projectType) {
   return { type: TOGGLE_GALLERY, projectType: projectType };
 }
 
-registerReducers({selectedGallery});
+/**
+ * Takes a list of projects and appends it to the existing list of
+ * projects of the specified type.
+ * @param {Array} projects A list of projects which are all older than the
+ * the current oldest project, newest first.
+ * @param {string} projectType The type of the projects being added.
+ * @returns {{projects: Array, projectType: string, type: string}}
+ */
+export function appendProjects(projects, projectType) {
+  return {type: APPEND_PROJECTS, projects, projectType};
+}
+
+export function prependProjects(projects, projectType) {
+  return {type: PREPEND_PROJECTS, projects, projectType};
+}
```

Comment:
heads up that `prependProjects` is something i pulled in from a follow-on pr to avoid a bunch of merge conflicts, and isn't actually being used yet.